cmd/login/runner.go

```diff
@@ -107,60 +108,10 @@ func (r *runner) tryToReuseExistingContext(ctx context.Context) error {
 	if err != nil {
 		return microerror.Mask(err)
 	}
-
 	currentContext := config.CurrentContext
-	kubeContextType := kubeconfig.GetKubeContextType(currentContext)
-
-	switch kubeContextType {
-	case kubeconfig.ContextTypeMC:
```

Comment:
I have removed the logic here and instead returned the `r.loginWithKubeContextName` function that already takes care.